src/Mutator/ProfileList.php

```diff
@@ -433,12 +433,12 @@ private function __construct()
     }
 
     /**
-     * @return string[]
+     * @return array<int, string>
```

Comment:
it is `string[]` here...

Reply:
There are places that depend on the keys being integers here.